tests/sentry/integrations/slack/test_event_endpoint.py

```diff
@@ -168,7 +168,7 @@ def test_valid_token(self):
         assert resp.status_code == 200, resp.content
         data = dict(parse_qsl(responses.calls[0].request.body))
         unfurls = json.loads(data["unfurls"])
-        issue_url = f"http://testserver/organizations/{self.org.slug}/issues/{group1.id}/bar/"
```

Comment:
@EvanPurkhiser guessing this changed because it's a BS URL?

Reply:
No this is changed because the order of how it decides which URL (when there are multiple of the "same" urls) it decides to unfurl.

Basically if you link two issue pages, but linked to different sections